### PR TITLE
CATROID-627 Change delete and compute button

### PR DESCRIPTION
**Pull request by adnaribo**
https://jira.catrob.at/browse/CATROID-627
Change of compute and delete button and also changing the space between brick and formula editor to white instead of app background color.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: catroid/src/main/res/drawable/formula_editor_compute_button.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  ~ GNU Affero General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Affero General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
```
Comment (ndrnour) [inline catroid/src/main/res/drawable/formula_editor_compute_button.xml:24]: please rename this file to `formula_editor_compute_button_selector`

File: catroid/src/main/res/drawable/formula_editor_delete_button.xml [hunk 2]
```diff
@@ -26,12 +26,12 @@
 
     <item android:state_pressed="false">
         <shape>
```
Comment (ndrnour) [inline catroid/src/main/res/drawable/formula_editor_delete_button.xml:28]: same, also please rename it to `formula_editor_delete_button_selector`

File: catroid/src/main/res/drawable/ic_input_delete.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<!--
+  ~ Catroid: An on-device visual programming system for Android devices
+  ~ Copyright (C) 2010-2020 The Catrobat Team
+  ~ (<http://developer.catrobat.org/credits>)
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
```
Comment (ndrnour) [inline catroid/src/main/res/drawable/ic_input_delete.xml:6]: since this is a vector, please add size in dp as a suffix to the name, `ic_input_delete_18dp`

**Review by juliajulie95 — APPROVED** [review]

**Comment by wslany** [comment]
Thanks!

**Pull request merged**